gensim/models/keyedvectors.py

```diff
@@ -818,7 +937,7 @@ def nbow(document):
         # Compute WMD.
         return emd(d1, d2, distance_matrix)
 
-    def most_similar_cosmul(self, positive=None, negative=None, topn=10):
+    def most_similar_cosmul(self, positive=None, negative=None, topn=10, restrict_vocab=None):
```

Comment:
why `restrict_vocab` needed here?

Reply:
Same as the case of 3CosAdd. We would like to consider the 300,000 first words not every word of the vocabulary.